fenix/app/messaging.fml.yaml

```diff
@@ -265,3 +272,5 @@ enums:
         description: A banner in the homescreen.
       notification:
         description: A local notification in the background, like a push notification.
+      fullscreen:
+        description: A fullscreen message.
```

Comment:
🔍 Let's have something more descriptive.  Maybe: "A full screen dialog that is intended to be disruptive."